src/features/user/PlanetCash/screens/CreateAccount.tsx

```diff
@@ -0,0 +1,49 @@
+import { ReactElement, useState, useEffect } from 'react';
+import CreateAccountForm from '../components/CreateAccountForm';
+import CenteredContainer from '../../../common/Layout/CenteredContainer';
+import { usePlanetCash } from '../../../common/Layout/PlanetCashContext';
+import { CountryType } from '../../../common/types/country';
+import i18next from '../../../../../i18n';
+
+const { useTranslation } = i18next;
+
+const initialAllowedCountries: CountryType[] = [
+  { code: 'DE', label: 'Germany', phone: '49' },
+  { code: 'ES', label: 'Spain', phone: '34' },
+  { code: 'US', label: 'United States', phone: '1' },
```

Comment:
Would be nice to have the labels translated with i18n. We have localized country names in `country.json` - all other places where country names are placed in the code only in English should be deprecated and hopefully not used in the UI, e.g.

```javascript
const { t } = useTranslation(['country']);
{ code: 'US', label: t('country:us'), phone: '1' },
```
Code not tested, maybe the const must be initialized more dynamically.

Reply:
We're not using label anywhere that is visible to the user. I've updated this to remove label from the initialCountries array.